other/materials_designer/create_point_defect.ipynb

```diff
@@ -10,17 +10,10 @@
     "<h2 style=\"color:green\">Usage</h2>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    DEFECT_CONFIGS = [
</pre><p>Put a link to the defect configuration explanation in made (perhaps not in comment, but add a sentence below 1.1)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Not addressed</p>

Reply:
<p>We should either update the class docstring to have approximate coordinate and site_id or put links to specific </p>

Reply:
Updated the docstring and point link to that line